sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -228,6 +229,8 @@ def test_biqquery_read_streaming_fail(self):
                                  r'source is not currently available'):
       p.run()
 
+  @pytest.mark.skipif(sys.version_info >= (3, 7),
+                      reason='TODO(BEAM-8095): Segfaults in Python 3.7')
```

Comment:
hmmm... this only segfaults when using pytest?   what happens if you disable xdist?

Reply:
I changed it to `no_xdist`. It still segfaults when using xdist or when attempting to run under a debugger in PyCharm.

Reply:
For problems with this test see: https://issues.apache.org/jira/browse/BEAM-8397?focusedCommentId=16958087&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16958087